common/lib/xmodule/xmodule/js/src/sequence/display.coffee

```diff
@@ -114,6 +114,8 @@ class @Sequence
 
       sequence_links = @content_container.find('a.seqnav')
       sequence_links.click @goto
+
+      @content_container.focus();
```

Comment:
@talbs @cptvitamin Sending focus to the content container causes the jumping. Taking Mark's advice and adding an empty container just above it to which we'll send focus instead.